src/@ionic-native/plugins/android-full-screen/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@angular/core';
+import { Cordova, Plugin } from '@ionic-native/core';
+
+/**
+ * @name Android Full Screen
+ * @description
+ * This plugin enables developers to offer users a true full screen experience in their Cordova and PhoneGap apps for Android.
+ *
+ * Using Android 4.0+, you can use true full screen in "lean mode", the way you see in apps like YouTube, expanding the app right to the edges of the screen, hiding the status and navigation bars until the user next interacts. This is ideally suited to video or cut-scene content.
+ *
+ * In Android 4.4+, however, you can now enter true full screen, fully interactive immersive mode. In this mode, your app will remain in true full screen until you choose otherwise; users can swipe down from the top of the screen to temporarily display the system UI.
+ *
+ * ```typescript
+ * import { AndroidFullScreen } from '@ionic-native/android-full-screen';
+ *
+ * constructor(private androidFullScreen: AndroidFullScreen) { }
+ *
+ * ...
+ *
+ * this.androidFullScreen.isImmersiveModeSupported()
+ *   .then(() => this.androidFullScreen.immersiveMode())
+ *   .catch((error: any) => console.log(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'AndroidFullScreen',
+  plugin: 'cordova-plugin-fullscreen',
+  pluginRef: 'AndroidFullScreen',
+  repo: 'https://github.com/mesmotronic/cordova-plugin-fullscreen',
+  platforms: ['Android']
+})
+@Injectable()
+export class AndroidFullScreen {
+  /**
+   * Is this plugin supported?
+   */
```

Comment:
Sure, sorry. I see both `@return` and `@returns` in other plugins - does this make any difference?